runtime/doc/lsp.txt

```diff
@@ -344,6 +344,8 @@ LspReferenceText          used for highlighting "text" references
 LspReferenceRead          used for highlighting "read" references
                                                           *hl-LspReferenceWrite*
 LspReferenceWrite         used for highlighting "write" references
+                                                              *hl-LspHoverRange*
+LspHoverRange             used for highlighting hover ranges
```

Comment:
alignment of the tag looks off

Reply:
Thanks- should they all end at col 80?

Reply:
oh I was thinking the tags should left-align on the same column. doesn't matter too much guess :P